### ISSUE TITLE
documentation incorrect

### ISSUE THREAD

**Issue by mjenove** [MicrosoftDocs/windows-itpro-docs#2015]
From what I observe, "The file will not grow beyond this size; when the limit is reached, old log entries are deleted to make room for the newly created ones." is not true; log entries just stop happening after the file has reached the set size. (Windows 10 Version 1803 (OS Build 17134.286))

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3621dedd-460e-2afd-e6d6-ac7a5d0d6911
* Version Independent ID: 298b96a7-17d2-6c1f-a2d1-81c24ad80e81
* Content: [Configure the Windows Defender Firewall Log (Windows 10)](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-firewall/configure-the-windows-firewall-log)
* Content Source: [windows/security/threat-protection/windows-firewall/configure-the-windows-firewall-log.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/windows-firewall/configure-the-windows-firewall-log.md)
* Product: **w10**
* GitHub Login: @brianlic-msft
* Microsoft Alias: **justinha**

**Comment by gvanem** [comment]
I also have troubles with getting anything in my `pfirewall.log` file using the horrid Management Console to set it up for logging. But I've created my own [program](https://github.com/gvanem/wsock-trace/#firewall-monitor) that listens for events from the *Windows Filtering Platform* (WFP). And after switching off my router's firewall, my program shows tons of activity from port-scanners. But not a single byte is written to the `pfirewall.log` file. 

I see many from these guys in Ukraine. Ref. the *SpamHaus Block List* at: 
https://www.spamhaus.org/sbl/query/SBL423676

**Comment by e0i** [comment]
@officedocsbot assign @e0i

**Comment by e0i** [comment]
@mjenove 

The editing team is investigating the issue and we'll follow up with documentation updates once the case is confirmed.

Thank you for taking the time to provide feedback.

**Comment by j0rt3g4** [comment]
Hi there I am unable to confirm this, I have always seen the logs overwritten with new data over the years. So based on my experience if you require assistance please contact a partner or support.

**Comment by e0i** [comment]
@mjenove Would you confirm with @j0rt3g4 's comments regarding the issue? We would like to follow-up. Thanks.

**Comment by j0rt3g4** [comment]
@e0i  any update on this?

**Comment by e0i** [comment]
@mjenove 

We are closing this issue as there hasn't been a response from your side regarding clarification on document updates.

Thank you for providing feedback.

**Comment by e0i** [comment]
@officedocsbot close

**Comment by MichaelFirsov** [comment]
I confirm - in Windows Server 2019 after reaching the (default) limit of 4MB the logging just stops. Any actions (changing the limit to a higher value, changing the log location...) change nothing - the logging does not work.
This may be cause NOT by the reaching the limit - I manually enabled the firewall logging on my other Windows Server 2019 and it stopped logging after ~1 minute. Similarly, I failed to make it work again by changing the log location, restarting and etc.
By the way, I failed to make Windows Firewall log anything on my Windows  10 1803:
1) Applying a GPO with the firewall configuration (log all packets, limit size, log location = default) has no effect on Windows 10 except the message in its local firewall's properties  saying "For your security, some settings are controlled by a Group Policy." - the pfirewall.log file even does not get created!
2) After configuring the logging settings manually (with default values) the log file does get created but is never populated with any log records.
